SmartDeviceLinkTests/DevAPISpecs/SDLMenuManagerSpec.m

```diff
@@ -212,14 +228,20 @@ @interface SDLMenuManager()
                 });
 
                 it(@"should immediately attempt to update without the image", ^{
-                    testManager.menuCells = @[textAndImageCell];
+                    testManager.menuCells = @[textAndImageCell, submenuImageCell];
                     
                     NSPredicate *addCommandPredicate = [NSPredicate predicateWithFormat:@"self isMemberOfClass: %@", [SDLAddCommand class]];
                     NSArray *add = [[mockConnectionManager.receivedRequests copy] filteredArrayUsingPredicate:addCommandPredicate];
                     SDLAddCommand *sentCommand = add.firstObject;
 
-                    expect(add).to(haveCount(1));
+                    NSPredicate *addSubmenuPredicate = [NSPredicate predicateWithFormat:@"self isMemberOfClass: %@", [SDLAddSubMenu class]];
+                    NSArray *submenu = [[mockConnectionManager.receivedRequests copy] filteredArrayUsingPredicate:addSubmenuPredicate];
+                    SDLAddSubMenu *sentSubmenu = submenu.firstObject;
+
+                    expect(add).to(haveCount(2));
```

Comment:
There are a few failed test cases in this spec

Reply:
Fixed